lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -1,4 +1,4 @@
-;(function (define) {
+(function(define) {
```

Comment:
BTW, @jzoldak, you're going above and beyond by doing a line-by-line review of this, especially given how broken GitHub is being about it - I haven't reviewed the code changes individually, and I don't think Andy was planning on doing that either. We might want to have a discussion about if it's really needed.